source/features/latest-tag-button.tsx

```diff
@@ -90,13 +90,11 @@ async function init(): Promise<false | void> {
 	});
 
 	const link = (
-		<a className="btn btn-sm btn-outline ml-2 flex-self-center css-truncate rgh-latest-tag-button" href={String(url)}>
+		<a className="btn btn-sm btn-outline ml-0 mr-2 flex-self-center v-align-top css-truncate rgh-latest-tag-button" href={String(url)}>
```

Comment:
`.v-align-top` is needed to cancel the `vertical-align: middle` that comes with `.btn` and causes a misalignment when both features are under the branches dropdown (like in the second screenshot).